contents/handbook/product/prioritizing-work-for-mature-products.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Prioritizing work for mature products
+sidebar: Handbook
+showTitle: true
+---
+
+There's no golden rule or perfect framework for prioritization. Here are a few things we should keep in mind when building mature products (products with substantial revenue and usage). 
+
+- **Always prioritize our ICP**
+    - As products mature, we start to get more and more users who are not strictly within our ICP. Building for these users is great - but only if we've satisfied the core needs of our company-wide ICP first.
+    - Should different products have different ICPs?
+      - The very first ICP should always be the same. However, some products will fill their needs sooner than others.
+- **Prevent churn before capturing growth**
+    - Not all churn is preventable - some customers' businesses don't make it. These customers are usually in our smallest revenue bucket and shouldn't really impact prioritization - unless the question is about how to make them more successful (which is a good question for sure, but not always relevant).
+    - Preventable churn usually comes from:
+      - Lots of bugs or particularly bad bugs
+      - Missing features that competitors offer
+      - Pricing
+      - Not proving value often enough (eg. lack of use because it doesn't fit into their workflow)
+    - Churn should be roughy equivalent across products - if yours is high, figure out why and fix it. Growth is less effective when you have a leaky bucket.
```

Comment:
I think this is not necessarily true, e.g. once we build the messaging product, I would expect churning from that is harder than churning from Replay. In the first case, you would have to rip out and replace your whole messaging infrastructure, whereas Replay is more of a nice to have you can turn off for a month to save money

Reply:
I somewhat agree... But that doesn't change the fact that products that churn more are going to be harder to sustain. I think that makes the case that we should try harder to control churn for those products.